src/helperFunctions/getTasks.ts

```diff
@@ -0,0 +1,10 @@
+import task from '@/interfaces/task.type';
+
+export const appendNewTasks = (
+  taskList: task[],
+  unAssignedTasks: task[]
+  ) :task[] => {
+    const oldTasksIds = taskList.map((Task: task) => Task.id);
+    const newTasks = unAssignedTasks.filter((Task: task) => !oldTasksIds.includes(Task.id));
+    return [...taskList, ...newTasks];
+};
```

Comment:
Let's move this function in the component itself.
Helper functions are the functions that we'll be using more often in multiple places.